client/src/store/new-request-module.ts

```diff
@@ -2248,10 +2247,12 @@ export class NewRequestModule extends VuexModule {
         headers: { 'Content-Type': 'application/json' }
       })
 
-      this.mutateNameRequest(response.data)
-      this.mutateDisplayedComponent('Success')
```

Comment:
I removed this from this "network layer" and added it to the code that called this method.

I also updated this code to check `response.data` (same as the other patch method).